azure-cli/ad/Add-OAuthUserImpersonationScope.ps1

```diff
@@ -0,0 +1,49 @@
+function Add-OAuthUserImpersonationScope(
+    [Parameter(Mandatory = $true)]
+    [string] $spnId
+) {
+  # import utility functions
+  . "$PSScriptRoot\..\utilities\deploy.naming.ps1"
+
+  $existingManifest = az rest `
+    --method Get `
+    --url https://graph.microsoft.com/v1.0/applications/$spnId `
+    --headers "Content-Type=application/json" `
+    | ConvertFrom-Json
+
+  Throw-WhenError -output $existingManifest
+
+  $existingPermissionScopes = $existingManifest.api.oauth2PermissionScopes
+  $existingScope = $existingPermissionScopes | Where-Object {$_.value -eq "user_impersonation"}
+
+  if ($null -eq $existingScope){
```

Comment:
nit: missing space after `)`
the whole document can be autoformatted in vscode using alt-shift-f

Reply:
Thanks for the tip. Will do that for the other scripts as well